x-pack/plugins/spaces/public/management/edit_space/manage_space_page.tsx

```diff
@@ -124,7 +128,7 @@ export class ManageSpacePage extends Component<Props, State> {
     }
 
     return (
-      <Fragment>
+      <>
         <EuiPageHeader
           bottomBorder
```

Comment:
Possible to remove this `bottomBorder` prop? Better matches the designs without it.